app/models/concerns/slug.rb

```diff
@@ -13,7 +13,7 @@ def slug_exists?
   def blank_slug
     return unless is_a?(Article) || is_a?(Page)
 
-    self.slug = nil if status.blank? || (status.present? && !published?)
+    self.slug = nil if publication_status.blank? || (publication_status.present? && !published?)
```

Comment:
when we add the column can we make the defaultt value `draft` and then have a nonnull restraint. that way gauranteed not to be blank

Reply:
Great idea!